src/main/osd/osd_elements.c

```diff
@@ -178,6 +179,9 @@
 
 #define FULL_CIRCLE 360
 #define EFFICIENCY_MINIMUM_SPEED_CM_S 100
+#define EFFFILTERTC 0.1f
```

Comment:
```suggestion
#define EFFICIENCY_CUTOFF_HZ 10
```
Replace the value 10 with the cutoff frequency you want to get.